lib/layer.js

```diff
@@ -48,6 +48,31 @@ function Layer(path, options, fn) {
 }
 
 /**
+ * Handle any request for the layer.
+ *
+ * @api private
+ */
+
+Layer.prototype._handle_any = function _handle_any(/* ...args, next */) {
+  var self = this
+  var next = arguments[arguments.length - 1]
+  var handle = this.handle
+  var result
+  var hasNextBeenCalled = function () { return next._currentLayer === self }
```

Comment:
Best case scenario the fix in #26 means we don't need this. But I'll definitely fix if we do.